version: '3'

services:
  jupyter:
    image: jupyter/datascience-notebook
    restart: on-failure
    volumes:
      - /mnt/home/belter/github/jupyter-note:/mnt/notebook
      - /etc/localtime:/etc/localtime:ro
    environment:
      - PYTHONUNBUFFERED=1
    ports:
      - 8888:8888
    command: /bin/bash -c "jupyter lab --no-browser --ip=0.0.0.0 --notebook-dir=/mnt/notebook"
